packages/nova-routing/lib/store.js

```diff
@@ -1,7 +1,7 @@
 import { createStore, combineReducers, applyMiddleware, compose } from 'redux';
 // import { routerMiddleware } from 'react-router-redux'
 
-import { Reducers }from 'meteor/nova:lib';
+import { Reducers, Middleware }from 'meteor/nova:lib';
```

Comment:
We'll deprecate  `nova:lib` and group everything in `nova:core` to minimize confusion in a near future. We have a foot in each camp atm, but hope to get rid of `nova:lib` asap!

Could you had to `Middleware(s)` to `nova:core`'s entrypoint please? 🙏 See: https://github.com/TelescopeJS/Telescope/blob/devel/packages/nova-core/lib/modules.js#L1-L3